Resources/Prototypes/Entities/Structures/Machines/microwave.yml

```diff
@@ -105,3 +107,21 @@
   - type: GuideHelp
     guides:
     - Chef
+
+- type: entity
+  id: SyndicateMicrowave
+  parent: KitchenMicrowave
+  name: donk co. microwave
+  description: So advanced, it can cook donk-pockets in a mere 2.5 seconds!
+  components:
+  - type: Microwave
+    cookTimeMultiplier: 0.5
```

Comment:
Not a complaint, but this doesn't seem to actually get applied on spawns, might be conflicting with the machine parts being used - should test if it works upstream.